config/manager/kustomization.yaml

```diff
@@ -4,5 +4,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/isolatedcontainers/sandboxed-containers-operator
-  newTag: 4.8.0
+  newName: quay.io/openshift_sandboxed_containers/openshift-sandboxed-containers-operator
+  newTag: 1.0-latest
```

Comment:
Is there a way to have multiple tags? Here, I might want to have both `1.0` and `latest`, where the next release would have `1.0.1` and `latest` (moving the `latest` tag away from the earlier build).

Reply:
> > Same comment as for #92 . Can VERSION be defaulted to latest ?
> 
> @bpradipt Do you mean replacing `1.0-lastest` with just `latest`? That seems consistent with usage for most containers.
@c3d yes

Reply:
why I choose 1.0-latest is: we have users of the release-4.6 and release-4.7 branch. we have release-4.8 which we should soon rename to release-1.0.0. and we have master. Would :latest relate to a build of the latest 4.6, 4.7, 4.8(1.0.0) or master?
latest build of master would make most sense but is also not intuitive in my opinion and 1.0-latest, 4.6-latest etc is more clear.
What do you think?